src/main/kotlin/calculate/EmptyCheckValidation.kt

```diff
@@ -0,0 +1,10 @@
+package calculate
+
+class EmptyCheckValidation : CalculateValidation {
+    override fun validate(input: String) {
+        val target = input.replace(" ", "")
+        if (target.isEmpty()) {
+            throw IllegalArgumentException("입력값이 empty이면 안됨!")
+        }
```

Comment:
Kotlin에서 제공하는 [require](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html), [check](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/check.html) 함수를 활용해서 아래와 같이 더 간단하게 변경해볼 수 있을 것 같아요. 😃
```suggestion
            require(input.isNotBlank()) { "입력값이 empty이면 안됨!" }
```

Reply:
오오 이런 기능이..!